.gitignore

```diff
@@ -1,40 +1,10 @@
-# Built application files
-*.apk
-*.ap_
-
-# Files for the ART/Dalvik VM
-*.dex
-
-# Java class files
-*.class
-
-# Generated files
-bin/
-gen/
-out/
-
-# Gradle files
-.gradle/
-build/
-
-# Local configuration file (sdk path, etc)
-local.properties
-
-# Proguard folder generated by Eclipse
-proguard/
-
-# Log Files
-*.log
-
-# Android Studio Navigation editor temp files
-.navigation/
-
-# Android Studio captures folder
-captures/
-
-# Intellij
 *.iml
-.idea/workspace.xml
-
-# Keystore files
-*.jks
+.gradle
+.idea
+/local.properties
+/.idea/workspace.xml
```

Comment:
You can remove these lines since **.idea** defined above already includes _workspace_ and _libraries_